src/components/modals/AccountModal.vue

```diff
@@ -92,15 +92,6 @@ export default defineComponent({
       );
     });
 
-    // WATCHERS
-    watch(
-      () => props.open,
-      () => {
-        data.step = null;
-        data.connectorsLimit = 3;
```

Comment:
is this important?

Reply:
This is the cleanup logic - its just trying to restore it to the initial state which is:
```
    // DATA
    const data = reactive({
      connectorsLimit: 3,
      step: null,
      path:
        'https://raw.githubusercontent.com/snapshot-labs/lock/master/connectors/assets'
    });
```